src/NUnitFramework/framework/Internal/Commands/OneTimeTearDownCommand.cs

```diff
@@ -71,6 +71,9 @@ public override TestResult Execute(TestExecutionContext context)
                 IDisposable disposable = context.TestObject as IDisposable;
                 if (disposable != null && Test is IDisposableFixture)
                     disposable.Dispose();
+
+                //Clear references to test objects to reduce memory usage
+                context.TestObject = null;
```

Comment:
Yes, the context definitely goes out of scope once the test (case or suite) for which it was created completes. My operating theory was that the existence of a double chain of refs (context and test) one formed top down and the other bottom up was creating circularity that kept everything alive. So If we can do without the fixture in the test, I wonder if everything will start being released as it goes out of scope.